buildmaster/old_new_porting_map.yml

```diff
@@ -0,0 +1,382 @@
+  journal: JHEP 1405 (2014) 068
+ATLAS_WCHARM_WP_DIFF_7TEV:
+  new_name: ATLAS_WCHARM_7TEV_WP-LEPTONETA
+  reference_arxiv: '1402.6263'
+  journal: JHEP 1405 (2014) 068
+
+ATLASZPT8TEVYDIST:
+  new_name: ATLAS_Z0J_8TEV_PTY
+  reference_arxiv: '1512.02192'
+  journal: Eur. Phys. J. C76 (2016) 291
+ATLASZPT8TEVMDIST:
+  new_name: ATLAS_Z0J_8TEV_PTM
+  reference_arxiv: '1512.02192'
+  journal: Eur. Phys. J. C76 (2016) 291
+ATLAS_DY_2D_8TEV_LOWMASS:
+  new_name: ATLAS_DY_8TEV_LOWMASSY
+  reference_arxiv: '1710.05167'
+  journal: JHEP 12 (2017) 059
+ATLAS_WM_JET_8TEV_PT:
+  new_name: ATLAS_WJ_JET_8TEV_WMPT
+  reference_arxiv: '1711.03296'
+  journal: JHEP 05 (2018) 077
+ATLAS_WP_JET_8TEV_PT:
+  new_name: ATLAS_WJ_JET_8TEV_WPPT
+  reference_arxiv: '1711.03296'
+  journal: JHEP 05 (2018) 077
+ATLASDY2D8TEV:
+  new_name: ATLAS_Z0_8TEV_2DMY
+  reference_arxiv: '1606.01736'
+  journal: JHEP 08 (2016) 009
+ATLAS_WZ_TOT_13TEV:
+  new_name: ATLAS_DY_13TEV_TOT
+  reference_arxiv: '1603.09222'
+ATLAS_W_TOT_13TEV:
+  new_name: ATLAS_WMWP_13TEV_TOT
+  reference_arxiv: '1603.09222'
+  journal: Phys. Lett. B759 (2016) 601
+ATLAS_Z_TOT_13TEV:
+  new_name: ATLAS_Z0_13TEV_TOT
+  reference_arxiv: '1603.09222'
+  journal: Phys. Lett. B759 (2016) 601
+ATLASPHT15_SF:
+  new_name: ATLAS_PH_13TEV_XSEC
+  reference_arxiv: '1701.06882'
+  journal: Phys. Lett. B770 (2017) 473
+ATLAS_SINGLETOP_TCH_DIFF_7TEV_TBAR_RAP_NORM:
+  new_name: ATLAS_SINGLETOP_7TEV_TBAR-RAP-NORM
+  reference_arxiv: '1406.7844'
+  journal: Phys. Rev. D 90 (2014) 112006
+ATLAS_SINGLETOP_TCH_DIFF_7TEV_T_RAP_NORM:
+  new_name: ATLAS_SINGLETOP_7TEV_T-RAP-NORM
+  reference_arxiv: '1406.7844'
+  journal: Phys. Rev. D 90 (2014) 112006
+ATLAS_SINGLETOP_TCH_DIFF_8TEV_TBAR_RAP_NORM:
+  new_name: ATLAS_SINGLETOP_8TEV_TBAR-RAP-NORM
+  reference_arxiv: '1702.02859'
+  journal: Eur. Phys. J. C 77 (2017) 531
+ATLAS_SINGLETOP_TCH_DIFF_8TEV_T_RAP_NORM:
+  new_name: ATLAS_SINGLETOP_8TEV_T-RAP-NORM
+  reference_arxiv: '1702.02859'
+  journal: Eur. Phys. J. C 77 (2017) 531
+ATLAS_SINGLETOP_TCH_R_13TEV:
+  new_name: ATLAS_SINGLETOP_13TEV_TCHAN-XSEC
+  reference_arxiv: '1609.03920'
+  journal: JHEP 04 (2017) 086
+ATLAS_SINGLETOP_TCH_R_7TEV:
+  new_name: ATLAS_SINGLETOP_7TEV_TCHAN-XSEC
+  reference_arxiv: '1406.7844'
+  journal: Phys. Rev. D 90 (2014) 112006
+ATLAS_SINGLETOP_TCH_R_8TEV:
+  new_name: ATLAS_SINGLETOP_8TEV_TCHAN-XSEC
+
+# FTDY (non dw/ missing)
+DYE605_dw_ite:
+  new_name: DYE605_Z0_38P8GEV_DW_PXSEC
+  reference_arxiv: ''
+  journal: Phys. Rev. D43 (1991) 2815
+DYE886R_dw_ite:
+  new_name: DYE866_Z0_800_RATIO_DW_PDXSECRATIO
+  reference_arxiv: hep-ex/0103030
+  reference_hepdata: 10.17182/hepdata.22184/t4
+  journal: Phys. Rev. D64 (2001) 052002
+DYE886P:
+  new_name: DYE866_Z0_800_PXSEC
+  reference_arxiv: hep-ex/0302019
+DYE906R_dw_ite:
+  new_name: DYE906_Z0_120GEV_DW_PDXSECRATIO
+  reference_arxiv: '2103.04024'
+  journal: Nature 590 (2021), no. 7847 561
+CHORUSNB:
+  new_name: CHORUS_CC_AVG98GEV_NBPB
```

Comment:
Mmm clearly I'm missing some recent development on the naming, because I wouldn't have expected such a name ... (and again this does not match the theory side https://github.com/NNPDF/pinecards/tree/master/CHORUS_CC_NB_PB_SIGMARED ....)

Reply:
The naming was supposed to be `<experiment>_<process>_<energy>_<extrastuff>_<observable>`

Sadly the `<energy>` doesn't really work for all of the datasets so I had to move to "average energy" but... of course there might be better naming...

Reply:
Mmm ... average is an option, but if there is not a fixed collision energy (such as CC DIS) and the energy field is non-optional, but may contain non-numeric characters, I would suggest `DYNAMIC` or `NOTFIXED` or similar ... no need to hide that fact ...